hal/armv7m/stm32/l4/stm32l4.c

```diff
@@ -367,13 +433,21 @@ void _stm32_pwrSetCPUVolt(u8 range)
 time_t _stm32_pwrEnterLPStop(time_t us)
 {
 	unsigned int t;
+	int useMsi = 0;
 
 	/* Set internal regulator to default range to further conserve power */
 	_stm32_pwrSetCPUVolt(1);
```

Comment:
High performance mode is selected to conserve power? We seem no need it to switch to HSI, but comment seems off.

Reply:
This disables internal LDO (so not really high performance, just higher core voltage) and was recommended (if I remember correctly) in the docs.

Reply:
Fixed comment